compiler/resolution/functionResolution.cpp

```diff
@@ -30,6 +30,7 @@
 #include "build.h"
 #include "caches.h"
 #include "callInfo.h"
+#include "CatchStmt.h"
```

Comment:
If functionResolution compiles/works without CatchSTmt.h, let's remove this #include.